### PR TITLE
Order Share, Payment Options, and Add Payment are connected and functional. Need to fix CSS.

### PR DESCRIPTION
**Pull request by scottiehansen**

### CODE REVIEW

File: src/server/controllers.ts [hunk 1]
```diff
@@ -112,14 +112,21 @@ async function getPaymentsByUserId(req, res) {
...
-  const { user_id } = req.params;
-  const { name, card_number, card_type, exp_date, cvv, zip_code } = req.body;
+  console.log('recieved request');
+  console.log('new line');
+  console.log(req.body)
+  console.log('new line');
+  const { name, card_number, card_type, exp_date, cvv, zip_code, user_id } = req.body.params;
+  console.log('new line');
+  console.log(req.body)
+  console.log('new line');
```
Comment (Lordnickim) [inline src/server/controllers.ts:122]: You still need the comments? might be good to remove later

File: src/client/components/OrderShare.jsx [hunk 2]
```diff
@@ -231,7 +234,7 @@ const OrderShare = () => {
               />
             </div>
           </Payment> :
-          <Payment>
+          <Payment onClick={() => history.push('/PaymentOptions')}>
             <div>
               Add Card
             </div>
```
Comment (Lordnickim) [inline src/client/components/OrderShare.jsx:240]: Is it possible to hook this with Austins component?

**Review by Lordnickim — APPROVED** [review]
good to me just take a look at a few comments i made

File: src/client/components/addPayment/NewPaymentPage.jsx [hunk 3]
```diff
@@ -43,17 +45,22 @@ export default function NewPaymentPage() {
...
       zip_code: Number(values.zipCode),
-      user_id: userId,
-    }}
+      user_id: userId
+    }})
       .then(res => {
-        console.log('successfully added payment!');
+        console.log(res.data[0]);
+        console.log(typeof values.newCCNumber)
+        console.log(typeof newCCType)
```
Comment (austinchadoyeon) [inline src/client/components/addPayment/NewPaymentPage.jsx:60]: maybe clean up console.logs?

**Review by austinchadoyeon — APPROVED** [review]

**Pull request merged**